File: qt/gen_qvarlengtharray.cpp
// +build ignore

#include <qvarlengtharray.h>
#include "gen_qvarlengtharray.h"
size_type QVLABaseBase_Capacity(const QVLABaseBase* self) {
	return self->capacity();
}
size_type QVLABaseBase_Size(const QVLABaseBase* self) {
	return self->size();
}
bool QVLABaseBase_Empty(const QVLABaseBase* self) {
	return self->empty();
}

File: qt/gen_qvarlengtharray.go
package qt

import (
	"unsafe"
)

type QVLABaseBase struct {
	h          uintptr
	isSubclass bool
}

func (this *QVLABaseBase) Capacity() size_type {
	xxxxxxxxx
}

func (this *QVLABaseBase) Size() size_type {
	xxxxxxxxx
}

func (this *QVLABaseBase) Empty() bool {
	return (bool)(QVLABaseBase_Empty(this.h))
}
